static/templates/subscription_type.handlebars

```diff
@@ -1,2 +1,5 @@
-{{#if invite_only}}{{t 'This is an <span class="icon-vector-lock"></span> <b>invite-only stream</b>. Only people who have been invited can access its content, but&nbsp;any&nbsp;member&nbsp;of&nbsp;the&nbsp;stream can&nbsp;invite&nbsp;others.' }}
-{{else}}{{t 'This is a <span class="icon-vector-globe"></span> <b>public stream</b>. Anybody in your organization can join.' }}{{/if}}
+{{#if invite_only}}
+	{{t 'This is an <span class="icon-vector-lock"></span> <b>invite-only stream</b>' }}. This is owned by {{ stream_owner }}. {{t 'Only people who have been invited can access its content, but&nbsp;any&nbsp;member&nbsp;of&nbsp;the&nbsp;stream can&nbsp;invite&nbsp;others.' }}
```

Comment:
This isn't working. I'm just seeing the string "This is owned by " in my dev VM with no owner specified.